Dockerfile.tests

```diff
@@ -11,4 +11,4 @@ RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
 	export PATH=$PATH:$HOME/.cargo/bin && \
 	scripts/init.sh && \
 	TRYBUILD=overwrite cargo test --workspace --locked --release --verbose --features runtime-benchmarks --manifest-path bin/node/cli/Cargo.toml && \
-	WASM_BUILD_NO_COLOR=1 SUBSTRATE_TEST_TIMEOUT=1 cargo test -p substrate-test-utils --release --verbose --locked -- --ignored timeout
+	WASM_BUILD_NO_COLOR=1 SUBSTRATE_TEST_TIMEOUT=1 cargo test -p substrate-test-utils --release --verbose --locked -- --ignored timeout
```

Comment:
Why do we change this line? Please revert changes

Reply:
I reverted in the latest commit.